app/admin/template.hbs

```diff
@@ -34,6 +34,11 @@
             Accounts
           {{/link-to}}
         {{/link-to}}
+        {{#link-to "admin.imports" tagName="li"}}
+          {{#link-to "admin.imports"}}
+            Daily Imports
```

Comment:
No, if Ryan wants it to be `Daily Imports` then let's leave it.  This has me concerned though.  We use 3 different terms for the same thing:

* `Ads` in our Database and Business Layer
* `Raw` in the API
* `Daily Imports` in the UI

What do you think about asking Maxim to change `raw` to `ads` in the API layer?  At least then we can get rid of the term `raw` altogether and we will be left with only one inconsistency.